dev-cpp/sourcetrail/sourcetrail-2018.3.55.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop gnome2-utils xdg-utils
+
+DESCRIPTION="A cross-platform source explorer for C/C++ and Java"
+HOMEPAGE="https://www.sourcetrail.com/"
+SRC_URI="https://www.sourcetrail.com/downloads/${PV}/linux/64bit -> ${P}.tar.gz"
+
+LICENSE="Sourcetrail || ( GPL-2 GPL-3 LGPL-3 ) FDL-1.3 BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="examples"
+
+DEPEND="dev-util/patchelf"
+
+RDEPEND="
+	dev-libs/openssl
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/libpng:1.2
+	media-libs/mesa
+	sys-libs/libselinux
```

Comment:
Does it really require libselinux unconditionally? This would suck for systems not using selinux, I suppose.

Reply:
Nope it doesn't, nice catch :) I had just blindly converted all the bundled libs to RDEPEND, but libselinux is not directly used, so I have changed this to be conditionally dependent on the selinux use flag.